src/target/riscv/riscv.c

```diff
@@ -1063,10 +1067,13 @@ static int maybe_add_trigger_t2_t6_for_wp(struct target *target,
 			ret = try_setup_chained_match_triggers(target, trigger, lt_1, ge_2);
 			if (ret != ERROR_TARGET_RESOURCE_NOT_AVAILABLE)
 				return ret;
+		} else {
+			LOG_TARGET_DEBUG(target, "LT+GE chained match triggers are disabled for watchpoints. "
+						 "Use set_wp_enable_ge_lt_trigger to enable it.");
```

Comment:
Please update the name of the command here, too.

Reply:
Addressed.